addons/account/models/account_reconcile_model.py

```diff
@@ -366,6 +366,7 @@ def _get_write_off_move_lines_dict(self, st_line, residual_balance):
                 'analytic_account_id': line.analytic_account_id.id,
                 'analytic_tag_ids': [(6, 0, line.analytic_tag_ids.ids)],
                 'reconcile_model_id': self.id,
+                'journal_id': line.journal_id.id,
```

Comment:
a bit weird, but is not the journal linked to the account.move and not the account.move.line?

Reply:
this is not directly set as an aml ("reconcile_model_id" is also not a aml field)

Reply:
so the dict is not meant to set the fields on an aml?

Reply:
Not directly.

Reply:
@jbw-odoo hum hum https://github.com/odoo/odoo/blob/master/addons/account/models/account_move.py#L3510
This is currently the create vals for an aml yes...

Reply:
Edit: the docstring of this method is wrong.. This is not used to create some amls